packages/publisher-dapp/src/Pages/Enroll/styles.js

```diff
@@ -0,0 +1,28 @@
+export const useStyles = MUITheme => ({
+	enrollMainContainer:{
+		padding: '0 11%',
+		'& > div':{
+			padding: '32px 0',
+			borderTopWidth: 1,
+			borderTopColor: MUITheme.palette.text.lightGray,
+			borderTopStyle: 'solid',
+			'&:last-of-type, &:first-of-type': { borderTop: 'none' },
+			'& > p':{
+				'&:first-of-type': { marginBottom: 15 },
+				'& span':{
```

Comment:
why do we need a span here. `try` customizing it with typography.
```
<Typography display=“inline” component=“span”>
```
If that doesn't work, we'll discuss